project/script_VMSMER.tcl

```diff
@@ -59,9 +59,13 @@ set modules_to_test {
 }
 # module_to_export must correspond to the default macros set at the top of the
 # test bench; otherwise, the C/RTL cosimulation will fail
-set module_to_export VMSMER_D1PHIA
+set module_to_export VMSMER_L1PHIA
 
-set CFLAGS {-std=c++11 -I../TrackletAlgorithm -I../TopFunctions/CombinedConfig}
+# create new project (deleting any existing one of same name)
+open_project -reset vmsmeRouter
```

Comment:
@aryd Since you open a common project for all solutions here, can the opening of the region-specific project be removed from this line?:
https://github.com/cms-L1TK/firmware-hls/blob/70f68873848bf5b1eec22e8c02306f0af39bdb5e/project/script_VMSMER.tcl#L81-L82

Reply:
To be consistent with other script_XX.tcl I did the opposite and removed the common project...